lib/action-sheet/index.js

```diff
@@ -13,7 +13,7 @@ var ActionSheet = React.createClass({
     this.setState({ active: active });
   },
   onBodyClick: function onBodyClick(e) {
-    this.setActiveState(false);
+    if (!e.target.dataset['keepActionSheetActive']) this.setActiveState(false);
```

Comment:
Well, I could do something like the original code that climbed up the dom tree looking for a certain class. Essentially the same approach, but DRYer.